src/core.js

```diff
@@ -1228,6 +1228,10 @@ define([
         }).on('parent-question-type-change', function (e) {
             _this.jstree("set_icon", e.childMug.ufid, e.childMug.getIcon());
         }).on('remove-question', function (e) {
+            var currentMug = _this.getCurrentlySelectedMug();
+            if (e.mug && e.mug.parentMug && e.mug.parentMug === currentMug) {
+                _this.displayMugProperties(currentMug);
+            }
```

Comment:
Not only `selectSomethingOrHideProperties`, but anything that calls `jstree("select_node", ...)` will likely call `displayMugProperties`.